src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
Please be sure that the value of each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

Reply:
@Zeraltz This is the only requested change 🧐

Reply:
@mateo951  Oh I see, I didnt understand because it was under changes haha, now I get it, thank you very much for your time!